.vscode/settings.json

```diff
@@ -87,5 +87,6 @@
     "WPILOG",
     "Xstance",
     "YUYV"
-  ]  
+  ],
+  "java.jdt.ls.vmargs": "-XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xmx2G -Xms100m -Xlog:disable"

```

Comment:
What does this do?

Reply:
No idea to be honest, I think my computer just messed something with the settings when I was working